src/main/java/hw/hw5/Obstacle.java

```diff
@@ -9,5 +9,5 @@ public Obstacle(String name, int value) {
         this.value = value;
     }
 
-    public abstract boolean overcome(Participant participant);
+    public abstract boolean overcome(Participant Participant);
```

Comment:
```suggestion
    public abstract boolean overcome(Participant participant);
```
було вірно - змінні з малої букви називаються